modules/monitoring-disabling-cross-project-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="disabling-cross-project-alerting-rules-for-user-defined-projects_{context}"]
+= Disabling cross-project alerting rules for user-defined projects
+
+Creating cross-project alerting rules for user-defined projects is enabled by default. Cluster administrators can disable the capability in the `cluster-monitoring-config` config map.
```

Comment:
The main benefit is to prevent user-defined monitoring from overloading the cluster monitoring stack. We have counter-measures in place (limitation of the number of concurrent queries, resource limits, etc.) but admins might be extra-cautious and prefer opting-out for the feature until they have validated it.
It can also work as an escape hatch if a buggy rule is configured and you don't have time to identify which one is the culprit.